VideoLocker/src/test/java/org/edx/mobile/test/ServerJsonDateAdapterFactoryTest.java

```diff
@@ -64,6 +64,15 @@ public void testParsingMicrosecondsDate() {
     }
 
     @Test
+    public void testParsingNull() {
+        String dateString = "null";
+        Gson gson = newGson();
+        Date date = gson.fromJson(dateString, Date.class);
+
+        assertEquals(null, date);
```

Comment:
You could have used the `assertNull()` convenience method here, which also provides a more specific failure message.